src/bin/tui/peers.rs

```diff
@@ -71,9 +92,22 @@ impl TableViewItem<PeerColumn> for PeerStats {
 	where
 		Self: Sized,
 	{
+		// Compares used bandwidth of two peers
+		fn cmp_used_bandwidth(curr: &PeerStats, other: &PeerStats) -> Ordering {
+			let curr_recv_bytes = curr.received_bytes.unwrap_or(0);
+			let curr_sent_bytes = curr.sent_bytes.unwrap_or(0);
+			let other_recv_bytes = other.received_bytes.unwrap_or(0);
+			let other_sent_bytes = other.sent_bytes.unwrap_or(0);
+
+			curr_recv_bytes
```

Comment:
Do I understand correctly that it sorts by `recv_bytes`? Why not by sum of send + recv

Reply:
Sorts by `recv_bytes` and in case `recv_bytes` are equal sorts by `sent_bytes` as well. But yes, I also think that comparing total number of bytes went in both direction seems more correct.